### PR TITLE
Release v5

### LINKED ISSUE #961: Add "engines" property in package.json
For better user experience, we should include the [engines](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines) field in our package.json files.

### LINKED ISSUE #992: Vite Plugin for Source Maps
Is there any documentation / plugin to upload source maps with Vite instead of webpack? I’m unsure how to use the GitHub action with Vite (unsure where source maps are uploaded, the minified url, minified file, etc).

Any help would be appreciated. Using Vue3 + Vite.

Front logo [Front conversations](https://app.frontapp.com/open/top_go3tt)

### PR DESCRIPTION
**Pull request by subzero10**
## Status
**READY**

## Description

Closes #961 .
Closes #992 (original [PR here](https://github.com/honeybadger-io/honeybadger-js/pull/1006)).

Notes:
- PR title check will fail because the title is not CC syntax. It's OK, for this PR we will not squash -> we want the separate commits.
- Do not SQUASH when merging!

### CODE REVIEW

**Review by stympy — DISMISSED** [review]
It looks good... will the changelog be updated separately?

File: packages/rollup-plugin/MIT-LICENSE [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Copyright (c) 2017 Honeybadger Industries LLC
+Copyright (c) 2016 Brandon Doran
```
Comment (subzero10) [inline packages/rollup-plugin/MIT-LICENSE:2]: Is this necessary?

**Comment by subzero10** [comment]
Thank you @BethanyBerkowitz!

I think we are ready to release v5!
I will go ahead with your approval @stympy @BethanyBerkowitz.

File: packages/rollup-plugin/MIT-LICENSE (hunk 1 above)
Comment (stympy) [reply packages/rollup-plugin/MIT-LICENSE:2]: I think so, but @joshuap is probably the best one to answer 🙂

Comment (BethanyBerkowitz) [reply packages/rollup-plugin/MIT-LICENSE:2]: I copied the file directly from another package, so yeah I definitely defer to others on this!

Comment (stympy) [reply packages/rollup-plugin/MIT-LICENSE:2]: Yeah, if we are using other people's code, then we need to include them in the copyright notice.

Comment (stympy) [reply packages/rollup-plugin/MIT-LICENSE:2]: _and_ it needs to have had the same (MIT) license originally :)

**Review by BethanyBerkowitz — APPROVED** [review]
🥳

**Pull request merged**